project/1.18/assets/tinkers-construct/tconstruct/book/tinkers_gadgetry/zh_cn/armor/shell.json

```
@@ -1,14 +1,14 @@
 {
   "tool": "tconstruct:slime_leggings",
   "text": [
-    { "text": "Slimeshells are one part defense, one part storage, granting the best of both shulker abilities." },
-    { "text": "To create these wonderous pants, pour enderslime on a shulker shell in a basin. The shell can later be repaired using chorus.", "paragraph": true }
+    { "text": "黏液壳既能拿来防御，也能拿来存储，充分的发挥了潜影壳的两种能力。" },
```

```suggestion
    { "text": "黏液壳既能拿来防御，也能拿来存储，充分的发挥了潜影贝的两种能力。" },
```
这个是116的写的就有问题吗？shulker是潜影贝，shulker shell才是潜影壳